tests/_utils/test_prepare

```diff
@@ -100,3 +100,141 @@ function dmctl_operate_source() {
         "\"result\": true" 2 \
         "\"source\": \"$source_id\"" 1
 }
+
+# use this func to run case
+# param 1: case name
+# param 2: task config name
+# param 3: init_table command
+# param 4: clean_table command
+# param 5: shard mode
+function run_case() {
+    case=$1
+    task_conf=$2
+    init_table_cmd=$3
+    clean_table_cmd=$4
+    shard_mode=$5
+
+    echo "[$(date)] <<<<<< start DM-${case} ${shard_mode} >>>>>>"
+
+    eval ${init_table_cmd}
+
+    truncate -s 0 $WORK_DIR/master/log/dm-master.log
+    truncate -s 0 $WORK_DIR/worker1/log/dm-worker.log
+    truncate -s 0 $WORK_DIR/worker2/log/dm-worker.log
+
+    run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+        "start-task $cur/conf/${task_conf}.yaml --remove-meta"
+
+    # make sure worker switch to syncer
```

Comment:
I meet some cases that one worker was delayed until we run a ddl. eg.
start-task --> worker1 start to dump --> ddl --> worker2 start to dump
than it will fail in load unit.
Is this normal?

Reply:
address in 6fbfcc7